docs/src/templates/pages/styleguide/css-utilities/css-rwd.hbs

```diff
@@ -0,0 +1,64 @@
+---
+title: Responsive helper classes
+docs: true
+
+navgroup: styleguide
+navsub: css-utilities
+navactive: css-rwd
+layout: components
+---
+<div class="sg-index">
+
+{{#markdown}}
+
+Responsive helper classes to show/hide content based on our media-queries
+
+`.u-showBelowNarrow`
```

Comment:
Would be good to add these as headings -  probably h2's but much it to the styles on the other pages.